tests/integration-tests/tests/storage/test_efs.py

```diff
@@ -313,12 +313,7 @@ def _check_efs_correctly_mounted_and_shared(
     all_mount_dirs, remote_command_executor, scheduler_commands, iam_authorizations, encryption_in_transits
 ):
     for i, mount_dir in enumerate(all_mount_dirs):
-        test_efs_correctly_mounted(
-            remote_command_executor,
-            mount_dir,
-            encryption_in_transits[i],
-            iam_authorizations[i],
-        )
+        test_efs_correctly_mounted(remote_command_executor, mount_dir, encryption_in_transits[i], iam_authorizations[i])
```

Comment:
whitespace change?

Reply:
Will change it back. Was some automatic, local linter .. Will revert this.